examples/doc.txt

```diff
@@ -1,5 +1,5 @@
 /**
-@defgroup examples Mesh4all Examples
+@defgroup examples Examples
```

Comment:
```suggestion
@defgroup examples Sample applications running on some of our custom boards
```

Reply:
I add as description a brief documentation parameter, Could this works to a brew description or should take the suggested and apply as title?